ompi/mpi/c/Makefile.am

```diff
@@ -103,7 +105,6 @@ libmpi_c_mpi_la_SOURCES = \
         cart_sub.c \
         close_port.c \
         comm_accept.c \
-        comm_c2f.c \
```

Comment:
I'm not sure I understand why we would not build these interfaces...?

Even if we didn't compile OMPI with Fortran support, is there a reason to *not* build these interfaces?  I understand that they can't be used in a meaningful way, but is that a reason to make those symbols not available?  E.g., I could imagine a software package's configure script testing for the presence of various `MPI_*_f2c` or `MPI_*_c2f` functions as a proxy for determining whether an MPI implementation has MPI-2 support or not.